scalableminds/webknossos
# Use LZ4 with WASM for volume saving/undo/redo

See #5970 and https://github.com/scalableminds/webknossos/pull/6652#issuecomment-1324805564 for context and numbers.

### URL of deployed dev instance (used for testing):
- https://lzwasm.webknossos.xyz/

### Steps to test:
- brush and use undo/redo
- brush and save and reload page
- everything should work as usual

### Issues:
- fixes #5970

------
